packages/trader/src/App/Components/Elements/SideMenu/side-menu-headers.jsx

```diff
@@ -12,23 +12,25 @@ const SubHeaderLayout = ({ children }) => <div className='side-menu__sub-header-
 
 const MainHeaders = ({ items, handleSubroute, active_title }) => (
     <>
-        {items.map(item => (
-            <div key={item.title} className='side-menu'>
-                <VerticalTabHeader
-                    item={item}
-                    className={classNames('side-menu__header', {
-                        'side-menu__header--active'  : item.title === active_title,
-                        'side-menu__header--disabled': item.is_disabled,
-                    })}
-                    onChange={handleSubroute}
-                >
-                    <Icon
-                        icon='IconChevronRight'
-                        className='side-menu__icon'
-                    />
-                </VerticalTabHeader>
-            </div>
-        ))}
+        {items
+            .filter(item => !item.is_hidden)
+            .map(item => (
+                <div key={item.title} className='side-menu'>
+                    <VerticalTabHeader
+                        item={item}
+                        className={classNames('side-menu__header', {
+                            'side-menu__header--active'  : item.title === active_title,
+                            'side-menu__header--disabled': item.is_disabled,
```

Comment:
we are still using is_disabled here but we've removed is_disabled in account.jsx?

Reply:
@negarn I just removed `is_disabled` for Verification menu in `accounts.jsx` but this line that you are referring is a general feature than can be used for disabling any side-menu item. Do you really want to remove it?

Reply:
@negarn I didn't rename anything. I just added a new property `is_hidden` for removing items that have this property. the `is_disabled` property is working like before. But I am new in this project and I don't know what you know already about structure and conventions. 
Do you have any suggestion for changing this? or keep what is done by far?

Reply:
is_disabled was removed here https://github.com/binary-com/deriv-app/pull/447/files#diff-1fdbec725ecad93ae1dc1a3b44812291L72 and used is_hidden instead, so I want to make sure that if anyone else is expected is_disabled it's updated there too.

Reply:
@negarn you are right but that was the my task to not showing verification menu (instead of disabling it)
I can keep both of them like this:
```js
menu_item.is_disabled = !is_high_risk_client;
menu_item.is_hidden = !is_high_risk_client;
```
but I think the `is_disabled` is extra and not doing anything when `is_hidden` is set already.
Do you want to keep both of them there in `accounts.jsx`?

Reply:
no, your change is fine :) just wanted to be sure that it was intended to keep both. thanks.